inscriptions/ISic000696.xml

```diff
@@ -198,19 +198,23 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#JH">
                 <ab>
-                    <lb n="1"/><name>Iulia</name> <name>Gamice</name>
-                    <lb n="2"/><expan><abbr>vix</abbr><ex>it</ex></expan> <expan><abbr>ann</abbr><ex>is</ex></expan> <num value="70">LXX</num> <expan><abbr>h</abbr><ex>ic</ex></expan> <expan><abbr>s</abbr><ex>ita</ex></expan> <expan><abbr>e</abbr><ex>st</ex></expan>
-                    <lb n="3"/><name>Hygia</name> <supplied reason="lost">fili</supplied>a
-                    <lb n="4"/>matri o<supplied reason="lost">pti</supplied>mae
+                    <lb n="0"/><gap reason="lost" extent="unknown" unit="line"/>
+                    <lb n="1"/><supplied reason="lost"><gap reason="lost" extent="unknown" unit="character"/><expan><abbr>proc</abbr><ex>uratori</ex></expan> f</supplied>a<unclear>m</unclear><supplied reason="lost"><expan><abbr>il</abbr><ex>iarum</ex></expan> <expan><abbr>glad</abbr><ex>iatoriarum</ex></expan></supplied>
+                    <lb n="2"/><supplied reason="lost">per <gap reason="lost" extent="unknown" unit="character"/></supplied> <expan><abbr>Sic</abbr><ex>iliam</ex></expan> Ae<supplied reason="lost"><expan><abbr>m</abbr><ex>iliam</ex></expan><gap reason="lost" extent="unknown" unit="character"/></supplied>
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/> Dalma<supplied reason="lost"><expan><abbr>t</abbr><ex>iam</ex></expan><gap reason="lost" extent="unknown" unit="character"/></supplied>
+                    <lb n="4"/><supplied reason="lost"><expan><abbr>proc</abbr><ex>uratori</ex></expan> Ludi Matuti</supplied>ni duc<supplied reason="lost"><expan><abbr>en</abbr><ex>ario</ex></expan> Ludi Magni</supplied>
```

Comment:
again, put the supplied around each individual word, and encode ducenario on the model of the examples above.